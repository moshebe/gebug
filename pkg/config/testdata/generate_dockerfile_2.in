name: my-app
output_binary: /app
build_command: go build -o {{.output_binary}}
build_dir: /src
runtime_image: golang:1.15.2
run_command: '{{.output_binary}}'
debugger_port: 40000
debugger_enabled: false
expose_ports:
  - 8080