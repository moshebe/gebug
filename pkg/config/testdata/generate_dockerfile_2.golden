FROM golang:1.15.2
RUN go get github.com/githubnemo/CompileDaemon
RUN go get github.com/go-delve/delve/cmd/dlv
WORKDIR /src
COPY . .

ENTRYPOINT CompileDaemon -log-prefix=false -build="go build -o /app" -build-dir="/src" -command="/app"